src/manageTimers.tsx

```diff
@@ -19,47 +20,41 @@ export default function Command() {
   const { push } = useNavigation();
 
   useEffect(() => {
-    setInterval(async () => {
-      await refreshTimers();
+    setInterval(() => {
+      setTimers(getTimers());
```

Comment:
You should call `refreshTimers` here to load custom timers as well - your current approach only loads running timers, not presets.